src/field/field3d.cxx

```diff
@@ -923,7 +923,7 @@ const Field3D lowPass(const Field3D &var, int zmax, int zmin, REGION rgn) {
 
   checkData(var);
   Mesh *localmesh = var.getMesh();
-  int ncz = localmesh->LocalNz;
+  int ncz = localmesh->zend + 1 - localmesh->zstart;
```

Comment:
Is it worth making mesh::LocalNz a function, calculating this to avoid problems with missing out `+1` and making the intent clearer?

Reply:
We could, but I'd pick a different name to avoid confusion with `LocalNx` and `LocalNy` (and I guess we'd want to introduce comparable functions for the x and y directions).